src/states/features/auth/authSlice.js

```diff
@@ -14,6 +15,8 @@ const initialState = {
 };
 
 const resetStates = {
+  email: false,
+  incorrectCred: false,
```

Comment:
The state being captured in these cases appears to be the "message". It is recommended to use the "message" variable consistently throughout the code to maintain clarity and consistency.